### PR TITLE
Fix dot height in default mode

### PR DESCRIPTION
**Pull request by sergio-costas**
Fix https://github.com/micheleg/dash-to-dock/issues/2160

### CODE REVIEW

**Comment by sergio-costas** [comment]
@vanvugt Fixed the dot height in Gnome 46, and confirmed that it doesn't break in Gnome 45.

**Comment by vanvugt** [comment]
Thanks. Testing it I see the dot is still too close to the icon. And reverting to scale 100% doesn't totally fix that.

Can you check to see how upstream does their spacing?

**Comment by vanvugt** [comment]
Maybe fixing https://github.com/micheleg/dash-to-dock/issues/2173 would change things? Or maybe it's not related.

**Comment by sergio-costas** [comment]
I see the dot exactly in the same place than in gnome 45...

Gnome45:
Captura desde 2024-03-25 16-55-50

Gnome46:
imagen

**Comment by sergio-costas** [comment]
But, of course, I don't use scaling... At which scale are you using your desktop? How can I take into account scaling in CSS?

**Comment by 3v1n0** [comment]
> How can I take into account scaling in CSS?

Css should handle it automatically.

You can simulate scaling in a nested GNOME shell easily though. Launches it with something like `MUTTER_DEBUG_DUMMY_MONITOR_SCALES=1` and  `MUTTER_DEBUG_DUMMY_MODE_SPECS="1920x1080:3840x2160:4096x2160:7680x4320:3840x2400:3200x1800:2650x1440:1024x768:800x600:1440x900:1680x1050:1680x1012"`

**Comment by vanvugt** [comment]
> I see the dot exactly in the same place than in gnome 45...
> 
> Gnome45: Captura desde 2024-03-25 16-55-50
> 
> Gnome46: imagen

That's not a good example because they are different icons (one's not square). Also the dot is touching the bottom in the second screenshot, but not the first. But all this may be off-topic so I'll take new screenshots...

**Comment by vanvugt** [comment]
OK my main concern is only visible at 200% today:

| gnome-shell | #2168 |
| --- | --- |  
| scale200-gnome-shell | scale200-pr2168 |

It's minor and maybe can be ignored while we wait for https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/3245

File: _stylesheet.scss [hunk 1]
```diff
@@ -261,6 +262,10 @@ $dock_style_modes: [null, shrink, extended, extended-shrink];
     background-color: $remark_color;
 }
 
+.dash-item-container .overview-tile .app-grid-running-dot {
+    margin-bottom: $dash_padding_gnome - 3px;
```
Comment (vanvugt) [inline _stylesheet.scss:266]: Changing 3px to 4px resolves my concern about the slightly insufficient spacing. With 4px it now looks the same as upstream gnome-shell. Would 4px work for you?

Comment (sergio-costas) [reply _stylesheet.scss:266]: No problem. I'll do the change.

Comment (sergio-costas) [reply _stylesheet.scss]: Done.

Comment (vanvugt) [reply _stylesheet.scss]: Thanks. The change should have been squashed into the first commit so I force-pushed the squash into your branch. I will now merge it...

**Pull request merged**